store/postgres/src/deployment_store.rs

```diff
@@ -1015,9 +1024,9 @@ impl DeploymentStore {
         // Unwrap: If we are reverting then the block ptr is not `None`.
         let block_ptr_from = Self::block_ptr_with_conn(&site.deployment, &conn)?.unwrap();
 
-        // Sanity check on block numbers
-        if block_ptr_from.number != block_ptr_to.number + 1 {
-            panic!("revert_block_operations must revert a single block only");
+        // Sanity check on revert to ensure we go backward only
+        if block_ptr_to.number >= block_ptr_from.number {
```

Comment:
I do find the assertion that a single block is reverted reassuring. Perhaps we could have two fns that share the implementation, but have different asserts for different uses?

Reply:
Considering that Near skips blocks this is necessary, so this can be kept as is.